### PR TITLE
Update components content structure

### PR DESCRIPTION
**Pull request by karypun**
I've reviewed the components and updated the content across all to make them consistent.

### CODE REVIEW

**Comment by karypun** [comment]
@daniel-ac-martin @RobinKnipe Two components called Status message and Status banner are very similar in name and may confuse people. I propose that 'Status banner' be renamed to 'Expandable banner' to describe the component function. It will also be applicable across other types of content and not just statuses. I manage to change the name on the file itself but not on the side navigation menu. Let me know what you think.

File: components/footer/spec/Footer.stories.mdx [hunk 1]
```diff
@@ -32,10 +32,10 @@ HODS' page footer.
 <Props of={ Footer } />
 
 
-## Stories
-### Standard
+## When to use this component
+Use the footer at the bottom of every page of your service.
```
Comment (daniel-ac-martin) [inline components/footer/spec/Footer.stories.mdx]: Internal service?

File: components/status-banner/spec/StatusBanner.stories.mdx [hunk 2]
```diff
@@ -13,9 +13,9 @@ import readMe from '../README.md';
   } }
 />
 
-# Status banner
+# Expandable banner
```
Comment (daniel-ac-martin) [inline components/status-banner/spec/StatusBanner.stories.mdx]: We'll need to rename some of the files.

And also the `package.json`.

Comment (daniel-ac-martin) [reply components/status-banner/spec/StatusBanner.stories.mdx]: I can sost that stuff though.

**Review by daniel-ac-martin — APPROVED** [review]
This now LGTM but someone else should also review my own changes.

**Comment by daniel-ac-martin** [comment]
@karypun: Let me know if you are happy with this.

Preview: https://60afe41c03809100c6b9c869--hods-poc.netlify.app/components

**Pull request merged**